uECC.h

```diff
@@ -35,6 +35,15 @@ faster somewhat faster, but increases the code size. */
     #define uECC_SQUARE_FUNC 0
 #endif
 
+/* uECC_VLI_NATIVE_LITTLE_ENDIAN - If enabled (defined as nonzero), this will switch to native
+little-endian format for *all* arrays passed in and out of the public API. This includes public 
+and private keys, shared secrets, signatures and message hashes. 
+Using this switch reduces the amount of call stack memory used by uECC, since less intermediate
+translations are required. Note that this will *only* work on native little-endian processors. */
```

Comment:
Please also add a note that to use this, all buffers passed to uECC functions must be word-aligned.